lib/bloc/account/account_bloc.dart

```diff
@@ -269,10 +269,10 @@ class AccountBloc extends Cubit<AccountState> with HydratedMixin {
     required int amountSats,
   }) async {
     _log.v("addInvoice: $description, $amountSats");
-    return await _breezLib.receivePayment(
-      amountSats: amountSats,
-      description: description,
-    );
+
+    final requestData = sdk.ReceivePaymentRequest(amountSats: amountSats, description: description);
+    final responseData = await _breezLib.receivePayment(reqData: requestData);
+    return responseData.lnInvoice;
```

Comment:
As part of **TODO:1**,
This method will return `ReceivePaymentResponse` and the Invoice dialog will make use of both the `LNInvoice` & `openingFeeMsat`.